### PR TITLE
chore: Fix prettier config in itwinui-css

### PR DESCRIPTION
**Pull request by mayank99**
Before #868, we were relying on `editorconfig` which sets a line length of 120. This is now set in prettier.

This will fix the unintentional formatting changes we saw in https://github.com/iTwin/iTwinUI/pull/935#discussion_r1057810187 and https://github.com/iTwin/iTwinUI/pull/860#discussion_r1060937554

Also noticed and fixed stylelint was missing from fixOnSave (bad merge in #868).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,7 +0,0 @@
-[*]
```
Comment (rohan-kadkol) [inline .editorconfig:1]: How does `.editorconfig` work? Do we have to install any [VSCode plugin](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)? The [documentation](https://editorconfig.org/#:~:text=To%20use%20EditorConfig%20with%20one%20of%20these%20editors%2C%20you%20will%20need%20to%20install%20a%20plugin.) mentions this, but `.vscode/extensions.json` doesn't include it, so not sure.

**Review by rohan-kadkol — COMMENTED** [review]
You mentioned something about investigating/changing the CI at the end of https://github.com/iTwin/iTwinUI/pull/935#discussion_r1057810187. A little confused what you meant by that. So just confirming if CI is working as expected.

File: .editorconfig (hunk 1 above)
Comment (mayank99) [reply .editorconfig:1]: editorconfig has been removed. previously, we were using it through prettier, not directly. see docs: https://prettier.io/docs/en/options.html#print-width

**Comment by mayank99** [comment]
> You mentioned something about investigating/changing the CI at the end of [#935 (comment)](https://github.com/iTwin/iTwinUI/pull/935#discussion_r1057810187). A little confused what you meant by that. So just confirming if CI is working as expected.

ah, i was referring to the copyright linter, which we run in CI here: https://github.com/iTwin/iTwinUI/blob/main/.github/workflows/build.yml#L90

will be fixed in #952

File: .editorconfig [hunk 2]
```diff
@@ -1,7 +0,0 @@
...
-end_of_line = lf
-insert_final_newline = true
-indent_size = 2
-indent_style = space
-max_line_length = 120
-quote_type = single
```
Comment (rohan-kadkol) [inline .editorconfig:7]: To confirm, are all these properties the default in prettier? Or do we need to override any of them in prettier to maintain consistent behavior?

Comment (mayank99) [reply .editorconfig:7]: some of them are default, and most of them we set in `prettier-config.js` and `pretter-scss-config.js`.

**Review by rohan-kadkol — APPROVED** [review]

**Pull request merged**